livedata/src/test/java/io/getstream/chat/android/livedata/PaginationTest.kt

```diff
@@ -162,6 +165,54 @@ class PaginationTest {
                     },
                     it
                 )
+            },
+            // last_updated is a computed field based on max(createdAt, lastMessageAt)
```

Comment:
`last_updated` is not a field on `Channel` Wee need to add it

Reply:
what do you think is best? want to add a property on the low level client that computes the max(createdAt, lastMessageAt)?

Reply:
i saw the other PR, nice